[test-triage] chip_sw_hmac_enc_irq

### Hierarchy of regression failure

Chip Level

### Failure Description

   - /workspace/mnt/repo_top/sw/device/tests/hmac_enc_irq_test
  WARNING: Target pattern parsing failed.
  Analyzing: 0 targets (0 packages loaded)
  ERROR: no such package 'sw/device/tests/hmac_enc_irq_test': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
   - /workspace/mnt/repo_top/sw/device/tests/hmac_enc_irq_test
  INFO: Elapsed time: 20.687s
  INFO: 0 processes.
  FAILED: Build did NOT complete successfully (0 packages loaded)
  FAILED: Build did NOT complete successfully (0 packages loaded)
  make: *** [/workspace/mnt/repo_top/hw/dv/tools/dvsim/sim.mk:73: sw_build] Error 1

### Steps to Reproduce

GH revision: [9c0b24ddb](https://github.com/lowrisc/opentitan/tree/9c0b24ddbeb5616bd8db661ec1395f2d4f326bc4)

util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i chip_sw_hmac_enc_irq --build-seed 2483310614 --fixed-seed 4088507876

### Tests with similar or related failures

_No response_

--- a/hw/top_earlgrey/dv/chip_sim_cfg.hjson
+++ b/hw/top_earlgrey/dv/chip_sim_cfg.hjson
@@ -780,12 +780,6 @@
       run_opts: ["+sw_test_timeout_ns=15000000","+rng_srate_value=30"]
     }
     {
-      name: chip_sw_hmac_enc_irq
-      uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["sw/device/tests:hmac_enc_irq_test:1"]
-      en_run_modes: ["sw_test_mode_test_rom"]
-    }
-    {
       name: chip_sw_hmac_enc
       uvm_test_seq: chip_sw_base_vseq
       sw_images: ["//sw/device/tests:hmac_enc_test:1"]

